python/dask_cudf/dask_cudf/backends.py

```diff
@@ -474,10 +474,9 @@ def from_dict(data, npartitions, orient="columns", **kwargs):
 
             if orient != "columns":
                 raise ValueError(f"orient={orient} is not supported")
```

Comment:
Cant we remove this check (and remove `orient` from the function signature) now that we have a proper `cudf.DataFrame.from_dict` implementation?

It should also be fair game to officially add `from_dict` to `dask_cudf`, and simply invoke that function here. E.g.

```python
        @staticmethod
        def from_dict(data, **kwargs):
            from dask_cudf import from_dict

            return from_cudf(data, **kwargs)
```

Reply:
Done, also made an upstream PR that is needed to support this change: https://github.com/dask/dask/pull/9628